diff --git a/app/streaming/input/keyboard.cpp b/app/streaming/input/keyboard.cpp
index 51b9091..1cafb61 100644
--- a/app/streaming/input/keyboard.cpp
+++ b/app/streaming/input/keyboard.cpp
@@ -358,6 +358,18 @@ void SdlInputHandler::handleKeyEvent(SDL_KeyboardEvent* event)
             case SDL_SCANCODE_AC_HOME:
                 keyCode = 0xAC;
                 break;
+            case SDL_SCANCODE_AUDIONEXT:
+                keyCode = 0xB0;
+                break;
+            case SDL_SCANCODE_AUDIOPREV:
+                keyCode = 0xB1;
+                break;
+            case SDL_SCANCODE_AUDIOSTOP:
+                keyCode = 0xB2;
+                break;
+            case SDL_SCANCODE_AUDIOPLAY:
+                keyCode = 0xB3;
+                break;
             case SDL_SCANCODE_SEMICOLON:
                 keyCode = 0xBA;
                 break;
